app/controllers/candidate_interface/references/email_address_controller.rb

```diff
@@ -46,7 +46,7 @@ def referee_email_address_param
       end
 
       def redirect_to_review_page_unless_reference_is_not_requested_or_email_bounced
-        unless @reference.present? && (@reference.not_requested_yet? || @reference.email_bounced?)
+        unless @reference.present? && (@reference.editable? || @reference.email_bounced?)
```

Comment:
Needs the policy instance here, looks like the cause of the test failures too.

Reply:
Thanks - updated!